Documentation/ApiOverview/Backend/AccessControl/PasswordReset/Index.rst

```diff
@@ -1,72 +1,97 @@
+To display the reset link on the backend login page, the following criteria must
+be met:
 
-*  The user has a password entered previously (used to indicate that no third-party login was used)
-*  The user has a valid email added to their user record
-*  The user is neither deleted nor disabled
-*  The email address is only used once among all Backend users of the instance
+*   The user has a password entered previously (indicating that no third-party
+    login has been used).
+*   The user has a valid email address added to their user record.
+*   The user is neither deleted nor disabled.
+*   The email address is used only once for all backend users of the instance.
 
-Once the user has entered their email address, an email is sent out with a link to set a new password, which needs to have a least eight characters.
-The link is valid for 2 hours, and a token is added to the link.
-If the password was provided correctly, it is updated for the user, and they can log in.
+Once the user has entered their email address, an email is sent with a link that
+allows to set a new password, which must consist of at least eight characters.
+The link is valid for 2 hours and a token is added to the link. If the password
+is entered correctly, it will be updated for the user and they can log in.
 
 Notes on security
 =================
 
-*  When having multiple users with the same email address, no reset functionality is provided.
-*  No information disclosure is built-in, so if the email address is not in the system, it is not disclosed to the outside.
-*  Rate limiting is activated for allowing three emails to be sent within 30 minutes per email address.
-*  Tokens are stored in the database but hashed again, just like the password.
-*  When a user has logged in successfully (e.g., because they remembered the password), the token is removed from the database, effectively invalidating all existing email links.
+*   When having multiple users with the same email address, no reset
+    functionality is provided.
+*   No information disclosure is built-in, so if the email address is not in the
+    system, it is not disclosed to the outside.
+*   Rate limiting is enabled so that three emails can be sent per email address
+    within 30 minutes.
+*   Tokens are stored in the database but hashed again just like the password.
+*   When a user has logged in successfully (for example, because they remembered
+    the password), the token is removed from the database, effectively
+    invalidating all existing email links.
 
 .. index:: Backend; passwordReset
 .. index:: Backend; passwordResetForAdmins
 
 Global configuration
 ====================
 
-The feature is active by default and can be deactivated entirely via the
+The feature is enabled by default and can be deactivated entirely via the
 system-wide configuration option:
 
-:php:`$GLOBALS['TYPO3_CONF_VARS']['BE']['passwordReset']`
+..  code-block:: php
 
```

Comment:
```suggestion
..   code-block:: php
    :caption: config/system/additional.php | typo3conf/system/additional.php

```

Reply:
Nope, that was by intend. It can be set in settings.php and additional.php. And via the GUI.